frontend/add-leads/add-edit-materials.css

```
@@ -0,0 +1,18 @@
+.horizontal {
+  display: flex;
+  padding: 2rem, 2rem, 2rem, 0;
```

Commas are unneeded here:

```suggestion
  padding: 2rem 2rem 2rem 0;
```